<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SelectFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ChannelType">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Y channel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB planar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB packed</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="HeightEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="WidthEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="HeightLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLabel" name="WidthLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ElementType">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>UINT8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UINT16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>INT16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>INT32</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="PathLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected Path </string>
    </property>
   </widget>
   <widget class="QLabel" name="ImageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>ImageLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBinary_show">
    <property name="title">
     <string>Binary show </string>
    </property>
   </widget>
   <addaction name="menuBinary_show"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
